### ISSUE TITLE
I created a Cheatsheet for all the available commands, current to release 1,1

### ISSUE THREAD

**Issue by iammeat** [eduardofilo/RG350_adam_image#23]
I like to have a sheet that allows me to learn the commands for any new image, so I created this Adam Image Cheetsheet for my reference.  I hope others find it useful and I'm happy to keep it updated with any new commands as releases come out.  Any feedback is appreciated.

[Adam Image Cheatsheet v1.1.pdf](https://github.com/eduardofilo/RG350_adam_image/files/7106417/Adam.Image.Cheatsheet.v1.1.pdf)

**Comment by iammeat** [comment]
Slight update.  v1.1a

[Adam Image Cheatsheet v1.1a.pdf](https://github.com/eduardofilo/RG350_adam_image/files/7109363/Adam.Image.Cheatsheet.v1.1a.pdf)

**Comment by eduardofilo** [comment]
🤩
Thank you so much. I am going to attach this to the image's documentation. Very kind.

**Comment by iammeat** [comment]
No problem @eduardofilo.  I'll keep it up to date for each release.

**Comment by iammeat** [comment]
I just noticed that every single mention of "Shortcut" is spelled "Shorcut"!  I'll update it when the next release comes out.

**Comment by iammeat** [comment]
Cheatsheet updated for new RetroArch/FBA/CPS2 keys and spelling error correct.

[Adam Image Cheatsheet v1.2.pdf](https://github.com/eduardofilo/RG350_adam_image/files/7150148/Adam.Image.Cheatsheet.v1.2.pdf)

**Comment by eduardofilo** [comment]
Great!
I have uploaded to repository and linked in the release.
If you prefer, you can send me PRs with this in order to appear as a contributor in the repository. I would like.

**Comment by iammeat** [comment]
Yes, I was thinking that would be better so you can close this issue off.  I'll clone and do a PR tomorrow.
